### PR TITLE
ONPREM-328 - Machine runner package installation 

### PR DESCRIPTION
**Pull request by circlery**
# Description
Makes machine-runner installation by package the default installation method, instead of manual installation. 

# Reasons
[ONPREM-328]

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

[ONPREM-328]: https://circleci.atlassian.net/browse/ONPREM-328?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: jekyll/_cci2/runner-manual-installation-linux.adoc
Comment (christian-stephen) [inline jekyll/_cci2/runner-manual-installation-linux.adoc:1]: This still looks like it refers to launch-agent. For manually installing the _new_ agent, we also only want instructions on downloading the binary directly from the S3 bucket.

Other setup like permissions and setting up the systemd unit aren't something we want to maintain going forward outside the packages.

**Review by christian-stephen — CHANGES_REQUESTED** [review]

File: jekyll/_cci2/runner-manual-installation-linux.adoc
Comment (christian-stephen) [reply jekyll/_cci2/runner-manual-installation-linux.adoc:1]: Actually, looks like Stuart's PR #8334 should have this all covered. We can remove this page from the PR.

Comment (christian-stephen) [reply jekyll/_cci2/runner-manual-installation-linux.adoc:1]: After some discussion, we will want to maintain the launch-agent installation instructions during the preview, but we should be really explicit about what is for launch-agent and what is for the unified machine runner.

**Comment by circlery** [comment]
closing for https://github.com/circleci/circleci-docs/pull/8342

**Pull request closed without merging**